<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addressSetting</class>
 <widget class="QWidget" name="addressSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>200</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frame{
/*frame是整个widget里面的容器，用于放置控件 整体拖动*/
/*背景图片*/
background-image: url(:/new/prefix1/addressSettingBackgrond.png);
/*边缘圆角半径设置*/
border: 1px solid rgba(0,191,255, 255);
border-radius:5px; 
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>380</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>380</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>380</width>
      <height>60</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTextEdit" name="display_screen">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>300</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>11</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#FFFAFA;/*字体颜色设置*/
background-color: rgba(255,255,255,0);
border-style:inset;/*背景颜色设置*/
border-radius:2px;/*边框圆角半径设置*/
border: 1px solid rgba(70,130,180, 80);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="reset1">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">/*登录按钮*/
QPushButton{
/*静置时设置*/
color:#FFFAFA;/*字体颜色设置*/
background-color:rgba(70,130,180, 0);/*背景颜色设置*/  
border-style:outset;/*背景颜色设置*/
border-radius:10px;/*边框圆角半径设置*/
border: 1px solid rgba(70,130,180, 80)
}

QPushButton:hover{
/*悬停时设置*/ /*由于与内部代码有冲突 已弃用*/
background-color:rgba(135,206,250, 255);
border-style:outset;/*边框设置*/
}
QPushButton:pressed{
/*按压时设置*/
background-color:rgba(173,216,230, 255);
/*背景颜色设置*/
border-style:inset;/*边框设置*/
}</string>
      </property>
      <property name="text">
       <string>重置</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="LineEdit1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>300</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>15</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"> /*文本输入框 账号输入*/
QLineEdit{
/*非悬浮时颜色设置*/
color:#FFFAFA;
border-style:outset;/*设置边框*/
background-color: rgba(255,255,255,0);
border-radius:2px;/*边框圆角半径设置*/
border: 1px solid rgba(70,130,180, 80)

}
QLineEdit::hover
{
/*悬浮时颜色设置*/
background-color:rgba(135,206,250, 255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>25</number>
      </property>
      <property name="placeholderText">
       <string>网址</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="confirm1">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">/*登录按钮*/
QPushButton{
/*静置时设置*/
color:#FFFAFA;/*字体颜色设置*/
background-color:rgba(70,130,180, 0);/*背景颜色设置*/  
border-style:outset;/*背景颜色设置*/
border-radius:10px;/*边框圆角半径设置*/
border: 1px solid rgba(70,130,180, 80)
}

QPushButton:hover{
/*悬停时设置*/ /*由于与内部代码有冲突 已弃用*/
background-color:rgba(135,206,250, 255);
border-style:outset;/*边框设置*/
}
QPushButton:pressed{
/*按压时设置*/
background-color:rgba(173,216,230, 255);
/*背景颜色设置*/
border-style:inset;/*边框设置*/
}</string>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>380</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>380</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>380</width>
      <height>60</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QTextEdit" name="display_screen_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>300</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>11</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#FFFAFA;/*字体颜色设置*/
background-color: rgba(255,255,255,0);
border-style:inset;/*背景颜色设置*/
border-radius:2px;/*边框圆角半径设置*/
border: 1px solid rgba(70,130,180, 80);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="reset2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">/*登录按钮*/
QPushButton{
/*静置时设置*/
color:#FFFAFA;/*字体颜色设置*/
background-color:rgba(70,130,180, 0);/*背景颜色设置*/  
border-style:outset;/*背景颜色设置*/
border-radius:10px;/*边框圆角半径设置*/
border: 1px solid rgba(70,130,180, 80)
}

QPushButton:hover{
/*悬停时设置*/ /*由于与内部代码有冲突 已弃用*/
background-color:rgba(135,206,250, 255);
border-style:outset;/*边框设置*/
}
QPushButton:pressed{
/*按压时设置*/
background-color:rgba(173,216,230, 255);
/*背景颜色设置*/
border-style:inset;/*边框设置*/
}</string>
      </property>
      <property name="text">
       <string>重置</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLineEdit" name="LineEdit_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>300</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>15</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"> /*文本输入框 账号输入*/
QLineEdit{
/*非悬浮时颜色设置*/
color:#FFFAFA;
border-style:outset;/*设置边框*/
background-color: rgba(255,255,255,0);
border-radius:2px;/*边框圆角半径设置*/
border: 1px solid rgba(70,130,180, 80)

}
QLineEdit::hover
{
/*悬浮时颜色设置*/
background-color:rgba(135,206,250, 255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>25</number>
      </property>
      <property name="placeholderText">
       <string>网址</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="confirm2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">/*登录按钮*/
QPushButton{
/*静置时设置*/
color:#FFFAFA;/*字体颜色设置*/
background-color:rgba(70,130,180, 0);/*背景颜色设置*/  
border-style:outset;/*背景颜色设置*/
border-radius:10px;/*边框圆角半径设置*/
border: 1px solid rgba(70,130,180, 80)
}

QPushButton:hover{
/*悬停时设置*/ /*由于与内部代码有冲突 已弃用*/
background-color:rgba(135,206,250, 255);
border-style:outset;/*边框设置*/
}
QPushButton:pressed{
/*按压时设置*/
background-color:rgba(173,216,230, 255);
/*背景颜色设置*/
border-style:inset;/*边框设置*/
}</string>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="close">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">/*关闭按钮*/</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../TFW_chatServe/Resource.qrc">:/new/prefix1/close.png</pixmap>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
